ci/pipelines/deploy-to-perf.yml

```diff
@@ -0,0 +1,231 @@
+  #   type: registry-image
+  #   icon: docker
+  #   source:
+  #     repository: govukpay/notifications
+  #     variant: perf
+  #     <<: *aws_test_config
+  - name: nginx-proxy-ecr-registry-perf
+    type: registry-image
+    icon: docker
+    source:
+      repository: govukpay/docker-nginx-proxy
+      variant: perf
+      <<: *aws_test_config
+  # - name: nginx-forward-proxy-ecr-registry-perf
+  #   type: registry-image
+  #   icon: docker
+  #   source:
+  #     repository: govukpay/nginx-forward-proxy
+  #     variant: perf
+  #     <<: *aws_test_config
+  - name: slack-notification
+    type: slack-notification
+    source:
+      url: https://hooks.slack.com/services/((slack-notification-secret))
+
+resource_types:
+  - name: registry-image
+    type: registry-image
+    source:
+      repository: concourse/registry-image-resource
+      tag: "1.4.1"
+  - name: slack-notification
+    type: docker-image
+    source:
+      repository: cfcommunity/slack-notification-resource
+      tag: latest
+
+groups:
+  - name: adminusers
+    jobs:
+      - deploy-adminusers-to-perf
+  - name: update-deploy-to-perf-pipeline
+    jobs:
+      - update-deploy-to-perf-pipeline
+
+jobs:
+  - name: update-deploy-to-perf-pipeline
+    plan:
+      - get: deploy-to-perf-pipeline-definition
+        trigger: true
+      - set_pipeline: deploy-to-perf
+        file: deploy-to-perf-pipeline-definition/ci/pipelines/deploy-to-perf.yml
+
+  - name: deploy-adminusers-to-perf
+    serial: true
+    plan:
+      - get: adminusers-ecr-registry-perf
+        trigger: true
+      - get: nginx-proxy-ecr-registry-perf
+        trigger: true
+      - get: telegraf-ecr-registry-perf
+        trigger: true
+      - get: pay-infra
+      - get: pay-ci
+      - load_var: application_image_tag
+        file: adminusers-ecr-registry-perf/tag
+      - load_var: nginx_image_tag
+        file: nginx-proxy-ecr-registry-perf/tag
+      - load_var: telegraf_image_tag
+        file: telegraf-ecr-registry-perf/tag
+      - task: create-notification-snippets
+        file: pay-ci/ci/tasks/create-notification-snippets.yml
+        params:
+          APP_NAME: adminusers
+          ACTION_NAME: Deployment
+          <<: *snippet_params_all_versions
+      - load_var: success_snippet
+        file: snippet/success
+      - load_var: failure_snippet
+        file: snippet/failure
+      - load_var: start_snippet
+        file: snippet/start
+      - <<: *put_start_slack_notification
+      - task: assume-role
+        file: pay-ci/ci/tasks/assume-role.yml
+        params:
+          AWS_ROLE_ARN: arn:aws:iam::((pay_aws_test_account_id)):role/concourse
+          AWS_ROLE_SESSION_NAME: terraform-test-assume-role
+      - load_var: role
+        file: assume-role/assume-role.json
+        format: json
+      - task: check-release-versions
+        file: pay-ci/ci/tasks/check-release-versions.yml
+        params:
+          <<: *check_release_versions_params
```

Comment:
Do we want to bother with the check_release_versions stuff? I think the perf environment is somewhere where we're likely to want to roll back/forward more regularly (with no ill-effects).

Reply:
I guess it does make sense we'd want to roll back/forward more regularly but in practice I think this will rarely happen. Make sense to remove this task in any case.